ethers-core/src/types/transaction/eip2718.rs

```diff
@@ -473,6 +550,12 @@ impl From<OptimismDepositedTransactionRequest> for TypedTransaction {
         TypedTransaction::OptimismDeposited(src)
     }
 }
+#[cfg(feature = "kroma")]
```

Comment:
Pleas new line above this.
```suggestion

#[cfg(feature = "kroma")]
```